src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetFundsIn.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE PROCEDURE [employer_financial].[GetFundsIn]
```

Comment:
It might be worth renaming the repo and sproc to match the type it returns i.e. `ILevyFundsInRepository` & `GetLevyFundsIn`.